shortfin/build_tools/build_linux_package.sh

```diff
@@ -40,6 +41,8 @@ ARCH="$(uname -m)"
 MANYLINUX_DOCKER_IMAGE="${MANYLINUX_DOCKER_IMAGE:-quay.io/pypa/manylinux_2_28_${ARCH}:latest}"
 PYTHON_VERSIONS="${OVERRIDE_PYTHON_VERSIONS:-cp311-cp311 cp312-cp312 cp313-cp313}"
 OUTPUT_DIR="${OUTPUT_DIR:-${THIS_DIR}/wheelhouse}"
+CACHE_DIR="${CACHE_DIR:-}"
+SHORTFIN_ENABLE_TRACING="${SHORTFIN_ENABLE_TRACING:-ON}"
```

Comment:
```suggestion
SHORTFIN_ENABLE_TRACING="${SHORTFIN_ENABLE_TRACING:-OFF}"
```
What this set to be `ON` or `OFF` by default? From the PR title and the modified lines 17 and 20 it seems we might want `OFF` as a default here.

Reply:
In my prototype https://github.com/nod-ai/shark-ai/pull/625 , I

* Set it explicitly to `ON` in `build_packages.yml` so release builds include tracing support
* Set it explicitly to `OFF` in `pkgci_build_packages.yml` so dev builds for CI integration tests are fast

For developers running the script manually, I like matching the release build behavior, not the "optimize for fast CI builds" behavior.